README.md

```diff
@@ -1,4 +1,4 @@
-# Parliament2
+# Parliament2 ![CI Badge](https://travis-ci.org/dnanexus/parliament2.svg?branch=master)
```

Comment:
Oh nice, I like how it's right next to the title.  I should do that for the agent